Telerik.JustMock/Expectations/CommonExpectation.cs

```diff
@@ -441,7 +442,8 @@ public IOccurrence InOrder(string message = null)
 		{
 			return ProfilerInterceptor.GuardInternal(() =>
 				{
-					this.behaviors.Add(new InOrderBehavior(this.Repository, this.Mock, message));
+                    (this as IMethodMock).IsInOrder = true;
```

Comment:
A direct cast is more appropriate for this case as there is the result from the "as" cast operator is not assigned to a variable.